02-electric/electric-section-two/electric-section-two.html

```diff
@@ -1,38 +1,100 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta http-equiv="X-UA-Compatible" content="IE=edge">
-  <meta name="viewport" content="width=device-width, initial-scale=1.0">
-  <link rel="stylesheet" href="../../00-overall/00-global-styles/global-styles.css">
-  <link rel="stylesheet" href="electric-section-two.css">
-  <title>01 - Electric - Section Two</title>
-</head>
-<body>
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link
+      rel="stylesheet"
+      href="../../00-overall/00-global-styles/global-styles.css"
+    />
+    <link rel="stylesheet" href="electric-section-two.css" />
+    <title>01 - Electric - Section Two</title>
+  </head>
+  <body>
+    <section class="electric-section-two-container">
+      <h1 class="electric-section-two-title"> Power In The Kitchen</h1>
```

Comment:
please nest everything one level deeper 
like the other components with a subcontainer (this will be helpful later)
```
<div class="electric-section-two-subcontainer>
  ~component contents~
</div>
```